checkpoint-interface.js

```diff
@@ -93,9 +93,10 @@ function _enterCpMode () {
   this._scratch = levelup('', {
     db: memdown
   })
-  this._getDBs.unshift(this._scratch)
+
+  this._getDBs = [this._scratch].concat(this._getDBs)
   this.__putDBs = this._putDBs
-  this._putDBs = [this._scratch]
+  this._putDBs = [this._scratch].concat(this._getDBs)
```

Comment:
don't know much about the code around this, but 've been reading the the related issues that led to this PR ( originally reading from the race conditions issue opened in ganache-cli repo ). 

as _putDBs and _getDBs seems to be different objects, is op ( @kammerdiener ) sure that this is not a typo ? 

Was `this._putDBs = [this._scratch].concat(this._putDBs);` intended instead ? 

( sorry if it is not, just trying to educate myself ^^" )